src/main/java/com/moorhuhnservice/moorhuhnservice/MoorhuhnServiceApplication.java

```diff
@@ -11,21 +11,18 @@
 @PropertySource(value = "classpath:db.properties")
 public class MoorhuhnServiceApplication {
 
-	public static void main(String[] args) {
-		SpringApplication.run(MoorhuhnServiceApplication.class, args);
-	}
-
-	@Bean
-	public WebMvcConfigurer corsConfigurer() {
-		return new WebMvcConfigurer() {
-			@Override
-			public void addCorsMappings(CorsRegistry registry) {
-				// allow CORS requests for all resources and HTTP methods from the frontend origin
-				registry.addMapping("/**")
-						.allowedMethods("OPTIONS", "HEAD", "GET", "PUT", "POST", "DELETE");
-			}
-		};
-	}
-
+  public static void main(String[] args) {
+    SpringApplication.run(MoorhuhnServiceApplication.class, args);
+  }
 
+  @Bean
+  public WebMvcConfigurer corsConfigurer() {
+    return new WebMvcConfigurer() {
+      @Override
+      public void addCorsMappings(CorsRegistry registry) {
+        // allow CORS requests for all resources and HTTP methods from the frontend origin
+        registry.addMapping("/**").allowedMethods("OPTIONS", "HEAD", "GET", "PUT", "POST", "DELETE");
+      }
+    };
+  }
 }
```

Comment:
pls try to remove commits where you dont change anything... its really bad for a reviewer to understand what are actuall changes...

Reply:
It is now formattet trough the prettier maven plugin. If i do not change anything it would not be commited?!

Reply:
ok